src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public enum Color {
+    WHITE,
+    BLACK,
+    RED,
+    GREEN,
+    BLUE,
+    PURPLE,
+    GREY,
+    YELLOW
```

Comment:
The enum constants are correctly named using uppercase letters, which is the standard convention for constants in Java. Good job following this convention.